server/conf/application.dev-browser-tests.conf

```diff
@@ -7,3 +7,6 @@ include "application.dev.conf"
 # they should enable them via the Feature Flags HTTP
 # handler as needed.
 application_status_tracking_enabled = false
+
+# Enable caching of JS/CSS resources to speed up tests.
+enable_asset_caching_for_browser_tests = true
```

Comment:
I like the general approach. Is there any way to enable the "fingerprinting alluded to  in the omment in DisableCachingFilter.java?

Setting a long cache definitely makes sense to me for CI / run-browser-tests. Is it possible that run-browser-tests-local would end up keeping this cached between runs? The answer may be likely not, but I don't have a lot of context here since as I understand, the spawned browser in this case is local to a machine (semi-persistent) and not ephemeral like our Docker containers.

Reply:
It kinda alludes to my question on the chat. I just tested and if we enable fingerprinting locally in dev mode - it automatically rebuilds/refingerprints JS upon changes. So we can potentially just enable it even in dev mode and enable full caching even on dev, which will help to avoid the need browser-test-specific logic.

@kmcnellis WDYT?

Reply:
I would be OK with that, as long as it doesn't increase compile time

Reply:
Opened another PR with the proposed approach: https://github.com/civiform/civiform/pull/3394/files It affects compile (recompile) time only when changing TS file and even than just 1-2 sec.